sampleFiles/clinical/immunotherapy.tsv

```diff
@@ -0,0 +1,2 @@
+program_id	submitter_donor_id	submitter_treatment_id	drug_rxnormcui	drug_name	immunotherapy_type
```

Comment:
since this therapy depends on rxnorm db it should be validated against it, I'm not sure if you checked that or not. similar to chemo therapy 
https://github.com/icgc-argo/argo-clinical/blob/develop/src/submission/submission-service.ts#L899-L904